### DISCUSSION TITLE
‹kfir/blog› | MDX Blog with Next.js 13

### DISCUSSION THREAD

**Discussion by giscus** [kfirfitousi/blog#30] [Comments]
# Kfir's Blog | MDX Blog with Next.js 13

A Proof of Concept for an MDX Blog built with Next.js 13

https://blog.kfirfitousi.com/posts/web-dev/mdx-nextjs-13

**Comment by MR-Addict** [comment]
Thanks for your work. It helps me a lot.

**Comment by an678-mhg** [comment]
10 điểm

**Reply by an678-mhg** [reply]
reply

**Reply by an678-mhg** [reply]
haha

**Comment by itzsrikanth** [comment]
Great job in putting this together, was very helpful!

**Comment by MrMaina100** [comment]
Up above you recommended the use of contentlayer but sadly content layer is not maintained anymore and their site has been down this week.
